test/extended/networking/util.go

```diff
@@ -203,7 +186,7 @@ func checkConnectivityToHost(f *e2e.Framework, nodeName string, podName string,
 	savedErr := err
 
 	// Debug
```

Comment:
This part doesn't work right any more and was only added to debug a long-ago test flake. You can just `return err` above and drop everything from here to the end of the function

Reply:
done